script.js

```diff
@@ -0,0 +1,25 @@
+const menu = document.querySelector('.mobile');
+const menuItems = document.querySelectorAll('.nav-tag');
+const hamburger = document.querySelector('.hamburger');
+const closeIcon = document.querySelector('.closeIcon');
+const menuIcon = document.querySelector('.menuIcon');
+const hamb = document.querySelector('.hamb');
+closeIcon.style.display = 'none';
+function toggleMenu() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.